packages/kbn-test/src/functional_test_runner/lib/docker_servers/README.md

```diff
@@ -0,0 +1,86 @@
+# Functional Test Runner - Docker Servers
+
+In order to make it simpler to run some services while the functional tests are running, we've added the ability to execute docker containers while the tests execute for the purpose of exposing services to the tests. These containers are expected to expose an application via a single HTTP port and live for the life of the tests. If the application exits for any reason before the tests complete the tests will abort.
+
+To configure docker servers in your FTR config add the `dockerServers` key to your config like so:
+
+```ts
+// import this helper to get TypeScript support for this section of the config
+import { defineDockerServersConfig } from '@kbn/test';
+
+export default function () {
+  return {
+    ...
+
+    dockerServers: defineDockerServersConfig({
+      // unique names are used in logging and to get the details of this server in the tests
+      helloWorld: {
+        /** disable this docker server unless the user sets some flag/env var */
+        enabled: !!process.env.SOME_APP_PORT,
+        /** the docker image to pull and run */
+        image: 'hello-world',
+        /** The port that this application will be accessible via locally */
+        port: process.env.SOME_APP_PORT,
+        /** The port that the container binds to in the container */
+        portInContainer: 8080,
+        /**
+         * OPTIONAL: string/regex to look for in the log, when specified the
+         * tests won't start until a line containing this string, or matching
+         * this expression is found.
+         */
+        waitForLogLine: 'hello',
```

Comment:
Duplicate line, I believe your intent was to keep the RegEx one.

Reply:
it's just to illustrate that this line takes both regexp and string values